include/nuttx/wireless/wireless.h

```diff
@@ -150,6 +150,42 @@
 #define SIOCSIWPTAPRIO      _WLIOC(0x0039)  /* Set PTA priority type */
 #define SIOCGIWPTAPRIO      _WLIOC(0x003a)  /* Get PTA priority type */
 
+/* -------------------- DEV PRIVATE IOCTL LIST -------------------- */
+
+/* These 32 ioctl are wireless device private, for 16 commands.
+ * Each driver is free to use them for whatever purpose it chooses,
+ * however the driver *must* export the description of those ioctls
+ * with SIOCGIWPRIV and *must* use arguments as defined below.
+ */
+
+#define SIOCIWFIRSTPRIV     _WLIOC(0x00e0)
+#define SIOCIWLASTPRIV      _WLIOC(0x00ff)
+
+/* Previously, we were using SIOCDEVPRIVATE, but we now have our
+ * separate range because of collisions with other tools such as
+ * 'mii-tool'.
+ * We now have 32 commands, so a bit more space ;-).
+ * Also, all 'even' commands are only usable by root and don't return the
+ * content of ifr/iwr to user (but you are not obliged to use the set/get
+ * convention, just use every other two command). More details in iwpriv.c.
+ * And I repeat : you are not forced to use them with iwpriv, but you
+ * must be compliant with it.
+ */
```

Comment:
This comment does not seem to be pertinent to this project, considering the references to `mii-tool` and `iwpriv.c`.

Reply:
Yes, let me remove the comment